styles/pov.h

```diff
@@ -4,14 +4,48 @@
 // TODO(hubbe): Make it work with FastLED
 #if defined(ENABLE_WS2811)
 
-// Usage: &style_pov or StylePtr<StylePtr>
-// return value: suitable for preset array
-
-// This style draws images in the air if you swing the saber
-// carefully back and forth. Unfortunately you can't really see
-// the result without a slow exposure camera. It's possible that
-// dotstar APA102 pixels would be fast enough to do the effect
-// full justice, but I haven't tried that.
+/* 
+This style draws images in the air if you swing the saber
+carefully back and forth. Unfortunately you can't really see
+the result without a slow exposure camera. It's possible that
+dotstar APA102 pixels would be fast enough to do the effect
+full justice, but I haven't tried that.
+
+Usage: &style_pov
```

Comment:
I think it might be better to move this one below and say something like:

In older files, you may see &style_pov used, which is essentially the same as StylePtr<StylePOV<>>()